### ISSUE TITLE
Druid spell Tranquility not scripted properly

### ISSUE THREAD

**Issue by peacebringer** [TeamEverlook/Everlook-BugTracker#270]
I finally got test for how long should visual effects last for druid spell Tranquility, they should last for full 10 seconds and not for 3-4 seconds like it's currently scripted on vmangos
That's kinda logical, visual effects should last as long as channeling spell time lasts (imagine that visual effects for mage spell blizzard to last 3 seconds while channeling time is way longer than that)

video proof recorded from classic era realm

https://streamable.com/aj9sni

**Comment by peacebringer** [comment]
I found video from 1.12.1 client too

Visual effects also lasted as long as channeling time lasted, it starts from 3:58

https://www.youtube.com/watch?v=08M1EuTR7eY

**Comment by red-candle** [comment]
Yeah, spells shouldn't be scripted at all - there are severe core issues. :(

**Comment by peacebringer** [comment]
> Yeah, spells shouldn't be scripted at all - there are severe core issues. :(

These bugs are already being fixed by Vmangos team.

I spoke with Everlook's developer Neto, he told me he will implement these fixes.
